geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml

```diff
@@ -37,7 +37,7 @@
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-jupyter_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-jupyter-leaflet_${scala.binary.version}</artifactId>
             <scope>runtime</scope>
```

Comment:
I was just trying to preserve the old behavior. Whether or not the leaflet library /should/ be included in ``geomesa-accumulo-spark-runtime`` by default is a separate question.

FYI, the only dependencies for ``geomesa-jupyter-leaflet`` are ``scala-library``, ``gt-data``,  ``commons-lang3``, and ``scala-xml_2.11``.